utm.yaml

```diff
@@ -983,9 +987,139 @@ components:
         operational intent may not create a conflict with a higher-priority operational intent.  A higher-priority
         operational intent may create a conflict with a lower-priority operational intent.  The regulator specifies
         whether an operational intent may create a conflict with other operational intents of the same priority.
+
+        The priority represents the type of flight and is required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION
+        (EU) 2021/664, paragraph 3. The following values are allowed:
+        - 0: Normal operation.
+        - 100: Special operations as defined in accordance with Article 4 of Implementing Regulation (EU) No 923/2012.
+      enum:
+      - 0
+      - 100
       type: integer
       default: 0
 
+    FlightAuthorisation:
+      description: |-
+        Flight authorisation of an operational intent. It includes mandatory information required by ANNEX IV of COMMISSION
```

Comment:
An "operational intent" is a thing described by the ASTM standard which we are using to communicate a "flight authorisation" which is defined by the regulation -- I'd recommend: `Full description of a flight authorisation including mandatory information required by ANNEX IV of COMMISSION...`.  No need to mention operational intent because this object schema is used in OperationalIntentDetails, so it's already pretty clear what it's attached to.